codespell_lib/data/dictionary.txt

```diff
@@ -10264,10 +10264,21 @@ enclosue->enclosure
 enclosung->enclosing
 enclude->include
 encluding->including
+encocde->encode
 encocded->encoded
+encocdes->encodes
+encocding->encoding
+encocdings->encodings
+enconde->encode
+enconded->encoded
+encondes->encodes
 enconding->encoding
 encondings->encodings
+encorde->encode, encore,
 encorded->encoded
+encordes->encodes
+encording->encoding
+encordings->encodings
 encorporating->incorporating
 encose->enclose
 encosed->enclosed
```

Comment:
I know it's not your change, but these, and the ones below, should probably also suggest encode*, given the s/d positioning on QWERTY keyboards.

Reply:
Good catch again, done in 6036275bfdff22c96381a7232759e4473fc9b134. Not sure if we should also add the following:

encosings->enclosings, encodings,

Reply:
Yeah nice spot!

Reply:
Added in fa0481780e35a11a3d01bbf8501b8bffb3cc68ff